docs/east_ocr.md

```diff
@@ -0,0 +1,113 @@
+#  Optical Character Recognition with Directed Acyclic Graph
+
+This document demonstrate how to create and use an Optical Character Recognition (OCR) pipeline based on [east-resnet50](https://github.com/argman/EAST) text detection model,
+[CRNN](https://github.com/MaybeShewill-CV/CRNN_Tensorflow) text recognition combined with a custom node implementation.
+
+Thanks to such pipeline, a single request to OVMS can perform a complex set of operations with a response containing
+recognized characters for all detected text boxes. 
+
+## OCR Graph
+
+Below is depicted the graph implementing a complete OCR pipelines. 
+
+![OCR graph](east_ocr.png)
```

Comment:
Custom east postprocessing node outputs are:
- text_images (shape: 1,N,C,H,W) FP32 (cropped, resized, grayscaled text images)
- text_coordinates (shape: 1,N,4) I32 (x1, y1, height and width integers)
- confidence_levels (shape: 1,N,1) FP32 (floating point scores)

Reply:
Also, the box under crnn node should be gray as well

Reply:
I'd rather use word "Gather" not "Collector" since json parameters are called "gather_from_node" and not "collect_from_node"